modules/gatsby/src/components/home.jsx

```diff
@@ -21,81 +21,59 @@
 import React, {Component} from 'react';
 import WebsiteConfigConsumer from './layout/website-config';
 import Hero from './hero';
+import GithubContributors from './github/github-contributors.jsx';
 
-export default class Home extends Component {
-  renderPage({config}) {
-    const {contributors = []} = this.props;
-
-    const {
-      HOME_HEADING,
-      HOME_RIGHT = '...',
-      HOME_BULLETS = [],
-      PROJECT_TYPE
-    } = config;
+function renderPage({config}) {
```

Comment:
Nits:
- When using functional components we still give them class names instead of function name, i.e nouns instead of verbs. E.g. `HomePage` instead of `renderPage`. 
- We do this because to the user it is an implementation detail how this component was implemented, not something we want to highlight by using different naming conventions.

Reply:
This is not a functional component.